src/Components/Admin/Assets/uploadAssetModal.js

```diff
@@ -96,31 +112,38 @@ export default function UploadAssetModal({
             default:
                 setFileType(FileTypes.NONE);
                 setErrors({
+                    ...errors,
                     fileType: "File type must be either GLTF or GLB",
                 });
         }
     };
 
     const handleUploadFileChange = (event) => {
+        setErrors({ ...errors, assetBlob: "" });
         if (event.target.files && event.target.files[0]) {
             const file = event.target.files[0];
             fileToByteArray(file, setAssetByteArray);
+
+            const name = file.name;
+            const lastDot = name.lastIndexOf(".");
+            const fileName = name.substring(0, lastDot);
+            setFileName(fileName);
```

Comment:
Personally I think it's fine to leave the extension with the file name here. But since you already did this, we can leave it.